freshmaker/pyxis_gql.py

```diff
@@ -374,7 +426,7 @@ def find_images_by_installed_rpms(
 
         :param list rpm_names: List of rpm names
         :param list content_sets: List of content sets
-        :param list repositories: List of repository paths
+        :param dict repositories: Dictionary of repository names as keys and repositories as values
```

Comment:
Please don't change this, this API is called by other code as well, or probably you were meant to update the docstring of `find_images_by_repository`?

Reply:
@qixiang I meant this function, because I think the repository argument is a dictionary, not a list

Reply:
the repository names are what needed by this API, so it's unnecessary to have a dict as the parameter.